cogs/vxtwitconverter/vxtwitconverter.py

```diff
@@ -0,0 +1,44 @@
+from redbot.core import commands
+from redbot.core.bot import Red
+from discord import Message
+from urlextract import URLExtract
+
+
+class VxTwitConverter(commands.Cog):
+    """Converts twitter link embeds to vxtwitter"""
+
+    def __init__(self, bot: Red):
+        super().__init__()
+        self.bot = bot
+
+    @commands.Cog.listener("on_message")
+    async def twit_replacer(self, message: Message):
+        # skips if the message is sent by the bot
+        if message.author == self.bot.user:
+            return
```

Comment:
I think it's better to ignore for all bots:
```suggestion
        if message.author.bot:
            return
```